source/grid/grid_in.cc

```diff
@@ -1777,7 +1777,7 @@ void GridIn<2>::read_netcdf (const std::string &filename)
   vertex_indices_var->get(&*vertex_indices.begin(), n_bquads, vertices_per_quad);
 
   for (unsigned int i=0; i<vertex_indices.size(); ++i)
-    AssertThrow(vertex_indices[i]>=0, ExcInternalError());
+    Assert(vertex_indices[i]>=0, ExcInternalError());
```

Comment:
I think this should also be an `AssertThrow`. We read `vertex_indices` from a file, so there is potential for things to just be wrong at run time. If you don't like `AssertThrow` with `ExcInternalError`, then maybe use a better error message via `ExcMessage`?